scripts/load_development_data

```diff
@@ -65,7 +62,11 @@ then
             download_development_images
         fi
         load_database_backup
+
+        echo "Running application database migrations"
+        source "${DIR}/update"
+        run_database_migrations
```

Comment:
I added this here, because it looks like the database dumps are always going to be slightly behind, and missing migrations, so this step seems to be always required after loading them in.